switch_mod/utilities.py

```diff
@@ -561,8 +561,10 @@ def _save_generic_results(instance, outdir):
             writer.writerow(['%s_%d' % (index_name, i + 1)
                              for i in xrange(var.index_set().dimen)] +
                             [var.name])
-            for key, v in var.iteritems():
-                writer.writerow(tuple(make_iterable(key)) + (v.value,))
+            # Write values sorted by indices
+            key_object_pairs = sorted(var.items())
+            for pair in key_object_pairs:
```

Comment:
Can you use:
for key, obj in key_object_pairs:
    writer.writerow(tuple(make_iterable(key)) + (obj.value,))

using pair[0] and pair[1] is less readable (in my opinion) and inconsistent with the rest of the codebase.